app/helpers/formatting_helper.rb

```diff
@@ -17,7 +16,7 @@ def ensure_sentence_ending(text)
   end
 
   def lowercase_first_letter(text)
-    return text if text.blank?
+    return text if text.blank? || text.split.first == text.split.first.upcase
 
```

Comment:
Changed this to not lowercase the first letter if the entire first word is capitalised. Previously "UPRN known" would have been changed to "uPRN known".